bridge/bridgeapp/views.py

```diff
@@ -13,6 +13,7 @@
 
 class BridgeHomeView(View):
     """Home URL view - Display 3 category options"""
```

Comment:
Just as a comment should primarily describe what the code does, a docstring should primarily identify what's the purpose of a class, function or method.

Every class, function or method has a body (that rarely is empty), which is where comments on the internals of a class, function or method should be made, not in the docstring.

So here the purpose is `"""view for the home page"""` or simeply `"""homepage view"""`

On line 18 should have a `"""method to process GET action"""`

Then above line 19 you describe what will be rendered, for example:
```
# render home,.html with category info and current year
```